src/handlers/glacier.rs

```diff
@@ -59,6 +61,47 @@ impl Handler for GlacierHandler {
     }
 }
 
-async fn handle_input(_ctx: &Context, _event: &Event, _cmd: GlacierCommand) -> anyhow::Result<()> {
+async fn handle_input(ctx: &Context, event: &Event, cmd: GlacierCommand) -> anyhow::Result<()> {
+    let is_team_member = if let Err(_) | Ok(false) = event.user().is_team_member(&ctx.github).await
+    {
+        false
+    } else {
+        true
+    };
+
+    if !is_team_member {
+        return Ok(())
+    };
+
+    let url = cmd.source;
+    let response = reqwest::get(&format!("{}{}", url.replace("github", "githubusercontent"), "/playground.rs")).await?;
```

Comment:
Use the `reqwest::Client` inside of the `GithubClient` using `GithubClient::raw`